### ISSUE TITLE
Test when evalmode is dynamic

### ISSUE THREAD

**Issue by lcylmhlcy** [kalviny/MSDNet-PyTorch#9]
Sorry, I couldn't understand the function `dynamic_evaluate(model, test_loader, val_loader, args)` when testing. 
```
if args.evalmode == 'anytime':
            validate(test_loader, model, criterion)
        else:
            dynamic_evaluate(model, test_loader, val_loader, args)
        return
```
I have obtained models on my dataset. I want to test the model's performance on some images in a dir. However, these images don't have labels. Could I use **dynamic (evalmode)** or just **anytime**? Does the test data need labels when chosing **dynamic**?

**Comment by kalviny** [comment]
To inference dynamically, you need to split one subset from those data with labels as the validation set to find the thresholds.

**Comment by lcylmhlcy** [comment]
Thank you. I know it.
